reacnetgenerator/reacnetgen.py

```diff
@@ -608,7 +608,7 @@ def _getatomeach(self):
                 key1 = np.array([int(x) for x in s[0].split(",")])
                 index = np.array(
                     [j for j in range(len(lineh)) if lineh[j] == "1"])
-                if(len(index)) > 0:
+                if index:
```

Comment:
Traceback (most recent call last):
  File "/home/travis/build/njzjz/reacnetgenerator/reacnetgenerator/test.py", line 36, in test_reacnetgen
    r.runanddraw()
  File "/home/travis/build/njzjz/reacnetgenerator/reacnetgenerator/reacnetgen.py", line 138, in runanddraw
    self.run()
  File "/home/travis/build/njzjz/reacnetgenerator/reacnetgenerator/reacnetgen.py", line 160, in run
    atomeach = self._getatomeach()
  File "/home/travis/build/njzjz/reacnetgenerator/reacnetgenerator/reacnetgen.py", line 611, in _getatomeach
    if index:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

Reply:
```suggestion
                if index.size:
```